README.md

```diff
@@ -66,4 +68,4 @@ Additional samples demonstrate various Fabric use cases and application patterns
 Hyperledger Project source code files are made available under the Apache
 License, Version 2.0 (Apache-2.0), located in the [LICENSE](LICENSE) file.
 Hyperledger Project documentation files are made available under the Creative
-Commons Attribution 4.0 International License (CC-BY-4.0), available at http://creativecommons.org/licenses/by/4.0/.
+Commons Attribution 4.0 International License (CC-BY-4.0), available at http://creativecommons.org/licenses/by/4.0/.
```

Comment:
Please, don't remove the newline at the end of the file.

Reply:
Updated